src/components/StartLink/index.css

```diff
@@ -0,0 +1,18 @@
+@import "../../custom-properties.css";
+
+/** @define StartLink; */
+
+.StartLink {
+  background: var(--green);
```

Comment:
Noticed when implementing https://github.com/barnardos/consent-form-builder/pull/327 that `background` here should really be `background-color`. We tend to prefer specific properties and on line 17 we use `background-color`.